<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTcpEdgeDeviceView</class>
 <widget class="QWidget" name="CTcpEdgeDeviceView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabDeviceWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>测试信息</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="1" column="2">
        <widget class="QTabWidget" name="tabDeviceBoxWidget">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>信息</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="CTcpEdgeCtrl_TextBox" name="m_boxInfo">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>错误原因汇总</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="CTcpEdgeCtrl_TextBox" name="m_boxErr">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QTableWidget" name="m_gridDataset">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>时间戳</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上下行</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>功能名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>功能ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>报文数据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>错误原因</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QTreeWidget" name="m_treeDataset">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>测试信息集合</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>报文监视</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="CTcpEdgeCtrl_TextBox" name="m_boxPkg">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTcpEdgeCtrl_TextBox</class>
   <extends>QTextEdit</extends>
   <header>Ctrl/TcpEdgeCtrl_TextBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
